src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -29,7 +28,7 @@ public QuestionsController(ILogger<QuestionsController> logger, GetQuestionQuery
     public async Task<IActionResult> GetQuestionById(string? section, [FromServices] ISubmitAnswerCommand submitAnswerCommand, CancellationToken cancellationToken)
     {
         var parameterQuestionPage = TempData[TempDataConstants.Questions] != null ? DeserialiseParameter<TempDataQuestions>(TempData[TempDataConstants.Questions]) : new TempDataQuestions();
-        string id = string.Empty;
+        string? id = string.Empty;
```

Comment:
IDE was complaining about null references on the line below with the `TempData["questionId"]` part so that made it quiet..

However I think this logic refactor reads a bit nicer: [1472724](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/288/commits/14727240f266b614ef9a8719cdced10e40d22ad6)